internal/static/i18n/en.yaml

```diff
@@ -59,6 +59,19 @@ Errors:
         NotExisting: Multifactor OTP (OneTimePassword) doesn't exist
         NotReady: Multifactor OTP (OneTimePassword) isn't ready
         InvalidCode: Invalid code
+      U2F:
+        NotExisting: U2F existiert nicht
```

Comment:
describe what u2f is?

Reply:
see comment above...
text is now in English